src/styles/searchResults.css

```diff
@@ -0,0 +1,35 @@
+.search-results {
+  align-items: center;
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  flex-direction: row;
```

Comment:
flex-direction: row is applied automatically, so its doesnt need to be declared

Reply:
Removed.